pkg/config/samples/default.dusk.toml

```diff
@@ -92,6 +93,11 @@ certFile=""
 # server TLS key file
 keyFile=""
 
+[rpc.rusk]
+
+# timeout for internal GRPC calls expressed in milliseconds
+timeout= 100
```

Comment:
100 ms may be much too short in some cases - think for instance the creation of a PLONK proof (known to be several seconds at the moment). I think between 5 and 10 seconds would be more sensible, though it would be even better if we can adjust this on a case-by-case basis, since this lengthy timeout is only useful in a limited amount of cases.

Reply:
ah right, I had in mind the full node (where timeouts should really be kept to a minimum). I will create a `contractTimeout` and a `defaultTimeout`. The former should be enough for transaction creations (which include PLONK), and the default should cover the rest. I don't think we need more granularity than that and we can always add parameters later

Reply:
Sounds good!